SetGame/CardsMatching.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CardsMatching.swift
+//  SetGame
+//
+//  Created by Artur Shellunts on 12.09.21.
+//
+
+import Foundation
+
+func valuesMatching<Type: Equatable>(_ value1: Type, _ value2: Type, _ value3: Type) -> Bool {
+	let allSameShape = value1 == value2 && value2 == value3
+	let allDifferentShape = value1 != value2 && value2 != value3 && value3 != value1
+
+	return allSameShape || allDifferentShape
+}
+
+func CardsMatching(_ card1: Card, _ card2: Card, _ card3: Card) -> Bool {
```

Comment:
Here one can read the rules: https://en.wikipedia.org/wiki/Set_(card_game).
I have not seen the word "match".
Probably one should find a better name for this from the domain language of the game.
Something with some words: "these three cards form a set".